lib/server.js

```diff
@@ -47,6 +48,12 @@ function createSocketIoServer (webServer, executor, config) {
   return server
 }
 
+function dieOnError (error) {
+  console.error(error)
```

Comment:
should it be `this.log.error` instead?

Reply:
@johnjbarton suggested here:
https://github.com/karma-runner/karma/pull/3039#discussion_r194440585

to use `console.error` to log fatal errors. I think it's ok.

Reply:
hmm... I suppose `this.log.error` was initially used here https://github.com/karma-runner/karma/pull/3039/files/193ccdd1147b58ee2117322a105155aea064827e#diff-c945a46d13b34fcaff544d966cffcabaL157

If we would like to replace `this.log.error` by `console.error` for fatal errors, it should be consistent across the module, see https://github.com/karma-runner/karma/pull/3039/files/193ccdd1147b58ee2117322a105155aea064827e#diff-c945a46d13b34fcaff544d966cffcabaR160

Otherwise I would leave the existing behavior in that PR.

Reply:
I've used `dieOnError` for that case now. Now use of `console.error` for fatal errors is consistent in this module.